web/docs/guides/websockets.md

```diff
@@ -4,67 +4,48 @@ title: WebSockets
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
 # WebSocket support
-Wasp provides a fully integrated WebSocket experience by utilizing [Socket.IO](https://socket.io/) on the client and server. We handle making sure your URLs are correctly setup, CORS is enabled, and provide a useful `useSocket` abstraction for use in React components.
+Wasp provides a fully integrated WebSocket experience by utilizing [Socket.IO](https://socket.io/) on the client and server. 
+
+We handle making sure your URLs are correctly setup, CORS is enabled, and provide a useful `useSocket` and `useSocketListener` abstractions for use in React components.
 
 To get started, you need to:
 1. Define your WebSocket logic on the server.
 2. Declare you are using WebSockets in your Wasp file, and connect it with your server logic.
 3. Use WebSockets on the client, in React, via `useSocket` and `useSocketListener`.
-4. Optionally define the types for messages for use on the client and server.
+4. Optionally, type the WebSocket events and payloads for full-stack type safety.
 
 We will cover all the steps above, but in an order that makes it easier to explain new concepts.
 
-## Enable in Wasp file
-Here, we specify we are using WebSockets by adding `webSocket` to our `app` and providing the required `fn`. You can optionally specify a list of `entities` for use in your context (like in Operations), as well as changing auto-connect behavior (on by default).
+## Turn on WebSockets in your Wasp file
+We specify that we are using WebSockets by adding `webSocket` to our `app` and providing the required `fn`. You can optionally specify a list of `entities` for use in your context (like in Operations), as well as changing the auto-connect behavior (on by default).
 
 ```wasp title=todoApp.wasp
 app todoApp {
   // ...
 
   webSocket: {
-    fn: import { webSocketFn } from "@server/webSocket.js"
-    // Optional: entities: [...]
-    // Optional: autoConnect: <bool>
-  },,
+    fn: import { webSocketFn } from "@server/webSocket.js",
+    entities: [], // optional
```

Comment:
I would consider just passing all the entities here automatically, and not having user to specify them. Because we have one global websocket, for whole app, so I don't see much use in explicitly mentioning which entities it uses.
If not now, we can possibly do it in the future, but I think it makes sense to do from the DX side.

Reply:
> I would consider just passing all the entities here automatically, and not having user to specify them. 

That's a good point, I remember us talking about it. 

I'll make the changes to receive all the available entities inside of context.